app/src/main/java/com/kickstarter/viewmodels/ProjectHolderViewModel.java

```diff
@@ -172,6 +176,7 @@ public ViewModel(final @NonNull Environment environment) {
       super(environment);
 
       this.ksCurrency = environment.ksCurrency();
+      this.isHorizontalRewardsEnabled.onNext(environment.enableHorizontalRewards().get());
```

Comment:
instead of emitting this value, we should have an output that describes the desired outcome. what's supposed to happen in the view if this is true or false? it's not clear

Reply:
This is used to hide the projectActionButtons on the viewHolder, and the logic is: 
```
ViewUtils.isLandscape(context()) && isHorizontalRewardsFeatureEnabled
```
So to create an output to emit just the boolean, we would need to have to context on the viewModel. Is there another way to do that?